app/components/views/Offset/index.tsx

```diff
@@ -464,54 +464,50 @@ export const Offset = (props: Props) => {
-              </Text>
-            </label>
-            <input
-              value={beneficiary}
-              onChange={(e) => {
-                setBeneficiary(e.target.value);
-              }}
-              placeholder={t({
-                id: "offset.who_beneficiary",
-                message: "To whom will this retirement be credited?",
-              })}
-            />
-          </div>
-          <div className={styles.input}>
-            <label>
-              <Text t="caption" color="lightest">
-                <Trans id="offset.beneficiary_address">
-                  BENEFICIARY ADDRESS (optional: defaults to connected address)
+          <div className={styles.beneficiary}>
+            <Text t="caption" color="lighter">
+              <Trans id="offset.retirement_message">
+                Who will this retirement be credited to?
+              </Trans>
+            </Text>
+            <div className={styles.input}>
+              <input
+                value={beneficiary}
+                onChange={(e) => setBeneficiary(e.target.value)}
+                placeholder={t({
+                  id: "offset.beneficiary",
+                  message: "Name or organisation",
+                })}
+              />
+            </div>
+
+            <div className={styles.input}>
+              <input
+                value={beneficiaryAddress}
+                onChange={(e) => setBeneficiaryAddress(e.target.value)}
+                placeholder={t({
+                  id: "offset.ethereum_address",
+                  message: "Ethereum address",
+                })}
+              />
+              <Text t="caption" color="lightest" className="defaultAddress">
+                <Trans id="offset.default_retirement_address_message">
+                  Defaults to the connected wallet address
                 </Trans>
               </Text>
-            </label>
-            <input
-              value={beneficiaryAddress}
-              onChange={(e) => {
-                setBeneficiaryAddress(e.target.value);
-              }}
-              placeholder={t({
-                id: "offset.which_address_retiring",
-                message: "Which address are you retiring on behalf of?",
-              })}
-            />
+            </div>
           </div>
+
           <div className={styles.input}>
             <label>
-              <Text t="caption" color="lightest">
-                <Trans id="offset.retirement_message">RETIREMENT MESSAGE</Trans>
+              <Text t="caption" color="lighter">
+                <Trans id="offset.retirement_message">Retirement message</Trans>
               </Text>
             </label>
             <textarea
               value={retirementMessage}
               onChange={(e) => {
-                if (e.target.value.length >= 280) {
-                  return;
-                }
+                if (e.target.value.length >= 280) return;
```

Comment:
This would be a change request and +1 day delay for re-review on my last team: "always use brackets for the return block"

....I'm just commenting to celebrate that I dgaf and it doesn't matter either way 🎉 🎉🎉 😛